### PR TITLE
Upgrade react and daisyui

### PR DESCRIPTION
**Pull request by Blakeinstein**
- **Implement carbon (squashed commit)**
- **revert carbon**

### CODE REVIEW

File: backend/indexer/indexer.py [hunk 1]
```diff
@@ -259,29 +259,6 @@ async def ingest_data_points(
             documents_to_be_upserted.append(chunk)
         logger.info("%s -> %s chunks", loaded_data_point.local_filepath, len(chunks))
 
-        # delete the file from temp dir after processing
```
Comment (S1LV3RJ1NX) [inline backend/indexer/indexer.py:262]: Why was this removed?

File: backend/indexer/indexer.py [hunk 2]
```diff
@@ -367,15 +344,9 @@ async def ingest_data(
                     batch_size=request.batch_size,
                 )
                 if pool:
-                    logger.info(
```
Comment (S1LV3RJ1NX) [inline backend/indexer/indexer.py:370]: Good to have these logging statements.

**Review by S1LV3RJ1NX — COMMENTED** [review]
minor doubts.

File: backend/indexer/indexer.py [hunk 3]
```diff
@@ -259,29 +259,6 @@ async def ingest_data_points(
...
-            if loaded_data_point.local_metadata_file_path:
-                os.remove(loaded_data_point.local_metadata_file_path)
-                logger.debug(
-                    f"Processing done! Deleting file {loaded_data_point.local_metadata_file_path}"
-                )
-        except Exception as e:
-            logger.exception(
-                f"Failed to delete file {loaded_data_point.local_metadata_file_path} after processing. Error: {e}"
-            )
-
```
Comment (S1LV3RJ1NX) [inline backend/indexer/indexer.py:284]: Why was this removed?

**Review by S1LV3RJ1NX — COMMENTED** [review]
updated the comment

File: backend/indexer/indexer.py (hunk 3 above)
Comment (Blakeinstein) [reply backend/indexer/indexer.py:284]: The temp directory from the library is auto-cleaned when the variable is out of scope. So, this code was redundant.

**Comment by sayan-truefoundry** [comment]
did we change the style of this input?
image

**Comment by sayan-truefoundry** [comment]
also i would suggest pls give the font color white to "Cancel"
image

**Comment by sayan-truefoundry** [comment]
create new data source flow has bugs now ... 
i can see after creating a valid data source using web, the drawer didn't get closed and after that i have hit the submit button again and a blank data source got created

**Comment by sayan-truefoundry** [comment]
in the embedded application the submit button looks faded now ... pls bring back the previous black color
image

**Comment by sayan-truefoundry** [comment]
image
 i will suggest make the text color white whenever this Cancel button is in hover state and it would be great if we could have the black background in Create button by default rather than this violet or indigio (we use in all other places)

**Comment by sayan-truefoundry** [comment]
in data source creation form, this now works fine and closes the drawer after successful data source creation that is fine.... but after that if i open the drawer again without refreshing then no tab is shown as selected in Data Source Type 
image

**Review by sayan-truefoundry — APPROVED** [review]
